data/database.sql

```diff
@@ -363,7 +363,7 @@ CREATE TABLE IF NOT EXISTS `site_scan_history` (
   CONSTRAINT `fk_site_scan_history_sites1`
     FOREIGN KEY (`sites_id`)
     REFERENCES `sites` (`id`)
-    ON DELETE NO ACTION
+    ON DELETE CASCADE
```

Comment:
Valid point, but if someone manually changes a primary key, something is really wrong.  It might be worth fixing this in the future, but because the rest of the relationships in the DB are set up this way, I think this should be addressed in another PR.